dto/scenarios.go

```diff
@@ -1,71 +1,76 @@
 package dto
 
 import (
+	"time"
+
 	"github.com/checkmarble/marble-backend/models"
+	"github.com/checkmarble/marble-backend/pure_utils"
+	"github.com/guregu/null/v5"
```

Comment:
Why do you need this ? I don't find the place where it's used

Reply:
Line 60. Basically I want to be able to distinguish between `{"user_id": null}` and `{ }` as far as the parsed user_id is concerned. AKA send null to empty it, send nothing to ignore it in the PATCH

Reply:
Incidentally I don't need it for the other field (a slice) because the zero value from the parsed string slice is nil, which gives a nil slice in the model, which we ignore in the repository - whereas an empty slice in the input gives a non-nil (empty) slice, which we can use to reset the field to []

Reply:
(it's basically the same as pgtype.text. As far as I can tell it's really the same thing and here I'm just choosing to use this rather than pgtype because the latter smells of "db", but same difference)